bp_be/src/include/bp_be_dcache/bp_be_dcache_pipeline.vh

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Name: 
+ *    bp_be_dcache_pipeline.vh
+ *
+ *  Description: 
+ *    Structs used for dcache pipeline
+ */
+
+`ifndef BP_BE_DCACHE_PIPELINE_VH
+`define BP_BE_DCACHE_PIPELINE_VH
+
+`define declare_bp_be_dcache_size_op_s                                             \
+  typedef struct packed {                                                          \
+    logic double_op;                                                               \
+    logic word_op;                                                                 \
+    logic half_op;                                                                 \
+    logic byte_op;                                                                 \
+  } bp_be_dcache_size_op_s;
+
+`define bp_be_dcache_size_op_width                                                 \
+  (1 + 1 + 1 + 1)
+
```

Comment:
Flatten size op into struct

Reply:
These also don't need to be parameterized, since they're constant width control signals